dat/missions/dvaered/dvaered-colonel-transport.lua

```diff
@@ -0,0 +1,142 @@
+   m(_([["Oh, and one more thing I've got to warn you about: another warlord would be only to happy to blow any of the colonel rank such as myself up, so you may need to expect attacks."]]))
+   vn.menu{
+      {_([["Remind me what system that's in?"]]), "what system"},
+      {_([["I'd be happy to do that!"]]), "sure"},
+      {_([["What is your name?"]]), "what is your name"},
+      {_([["Wait, why is there someone trying to kill you?"]]), "why would you die"},
+   }
+
+   vn.label("what system")
+   m(fmt.f([["Umm..." the Dvaered says. They consult a watch. "Alright, {pnt} is in {sys}."]]))
+     {pnt=mem.dest_planet, sys=mem.dest_sys}
+   vn.jump("choice")
+
+   vn.label("what is your name")
+   m(_([[The Dvaered seems slightly taken aback. "Well... that may also be classified information... call me Radver."]]))
+   vn.jump("choice")
+
+   vn.label("sure")
+   m(_([["Wonderful! I'll be on your ship when you leave."]]))
+   vn.func( function ()
+      accepted = true
+   end )
+
+   vn.run()
+
+   vn.label("why would you die")
+   m(_([[The Dvaered colonel looks uncomfortable. "I'm afraid that too is classified information. Let's just say that I've got a vendetta (not the ship, the relation) with this other warlord, and we're on blowing-each-other-up terms. Don't ask me to reveal more information."]]))
+
+   vn.label("choice")
+   m(_([["Well? Will you do this?"]]))
+   vn.menu{
+      {_([["Yep, I'd be glad to!"]]), "sure"},
+      {_([["Not going to happen, sorry."]]), "never"},
+   }
+
+   vn.label("never")
+   m(_([["That's sad. Oh well, I'll ask someone else." The Dvaered leaves.]]))
+   vn.done()
+
+   if not accepted then return end
+
+   misn.accept()
+
+   misn.setReward(750000)
+   misn.setDesc(fmt.f(_("Escort a Dvaered colonel, who is flying an Arsenal, to {pnt} in the {sys} system. You haven't been told why, but there may be a large payment."), {pnt=mem.dest_planet, sys=mem.dest_sys}))
+   misn.osdCreate(_("Dvaered colonel escort"), {
+      fmt.f(_("Escort a Dvaered colonel to {pnt} in the {sys} system.")), {pnt=mem.dest_planet, sys=mem.dest_sys},
+   })
+   misn.markerAdd( mem.dest_planet )
+   hook.land( "land" )
+   local colonel_ship = ship.get("Dvaered Arsenal")
+   escort.init ( colonel_ship, {
+         pilot.add( "Dvaered Arsenal", "ffriendly", source_system, "Radver" ) -- The Dvaered colonel is telling you to call them the name of the Arsenal; their real name is classified information.
+   })
+
+   hook.enter( "ambush" )
+end
+
+function ambush ()
+   if not naev.claimTest( system.cur(), true ) then return end
+   local dvaered_factions = faction.get("Dvaered")
+
+   fhostile = faction.dynAdd( dvaered_factions, "warlords_hostile", _("Warlords") )
+   ffriendly = faction.dynAdd( dvaered_factions, "warlords_friendly", _("Warlords") ) -- codespell:ignore ffriendly
+   faction.dynEnemy( ffriendly, fhostile ) -- codespell:ignore ffriendly
+
+   pilot.add( "Dvaered Phalanx", "fhostile", source_system, "Asheron Anomaly" )
```

Comment:
Isn't that field for the name of the ship?
The ship should be called "Asheron Anomaly". It doesn't mean anything.

Reply:
OK, you should then add a translatable mark: `_("Asheron Anomaly")`. However, I think it may be a bit confusing unless it is mentioned by Radver. Maybe `_("Warlord Lackey")` might be more easy to understand.